lte/gateway/c/session_manager/RuleStore.cpp

```diff
@@ -137,6 +137,20 @@ bool PolicyRuleBiMap::get_rule(
   return true;
 }
 
+bool PolicyRuleBiMap::get_rules(
```

Comment:
To differentiate between get rule name, and get its definition, wouldn't it be better `get_rules_definition`?  

Or maybe we already have a clear definition between functions that get rule name only and functions that gets its definition

Reply:
yeah to be extra clear I can rename it to `get_rules_by_ids`